pages/about.html

```diff
@@ -4,9 +4,64 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me!</title>
+  <link href="../styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <header class="social-links">
+    <a href="https://www.instagram.com/jessica.w.roberts/">
+    <img src="../images/Instagram.png" alt="Instagram Logo" height=25 width=25>
+    </a>
+    <a href="https://www.linkedin.com/in/jessicawroberts/">
+      <img src="../images/Linkedin.png" alt="Linkedin Logo" height=25 width=25>
+      </a>
+  </header>
+  <nav>
+    <ul class="main-nav">
+      <li><a href="index.html">home</a></li>
+      <li class="hash">/</li>
+      <li><a href="about.html">about</a></li>
+      <li class="hash">/</li>
+      <li><a href="portfolio.html">portfolio</a></li>
+      <li class="hash">/</li>
+      <li><a href="musings.html">musings</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h1>Hello, nice to meet you! I'm Jessica and I'm learning HTML & CSS.</h1>
+    <h2>Here's some additional things about me:</h2>
+    <section class="block-pics">
+      <figure class="likes">
```

Comment:
Since you put the `likes` class on every child it's clearer to just put it on the parent and use a child selector:

```suggestion
    <section class="block-pics likes">
      <figure>
```

```css
/* instead of:
.likes {
    /* Styles *​/
}
*/

.likes figure {
    /* Styles */
}
```